web/webhook.go

```diff
@@ -30,6 +29,15 @@ func incomingWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 	var err *model.AppError
 	incomingWebhookPayload := &model.IncomingWebhookRequest{}
 	contentType := r.Header.Get("Content-Type")
+
+	defer func() {
+		if *c.App.Config().LogSettings.EnableWebhookDebugging {
+			if err != nil {
+				mlog.Debug("Incoming webhook received", mlog.String("Id", id), mlog.String("Request ID", c.App.RequestId), mlog.String("Payload", incomingWebhookPayload.ToJson()))
```

Comment:
Our convention is to use lower snakecase for the keys. So `webhook_id`, `request_id`, `payload` please.

Reply:
@crspeller Thanks very much for the feedback. I addressed your comments and pushed a new commit. Please let me know if you see any other areas that need to be addressed.